### PR TITLE
requested changes on fake data and ACCEPTED response

### PR DESCRIPTION
**Pull request by Fareedh-12**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Comment by Fareedh-12** [comment]
I made the requested changes on the fake data and the accepted response page

File: client/src/pages/ViewRequestStatus/RequestStatusView.js [hunk 1]
```diff
@@ -55,14 +55,11 @@ export function RequestStatus() {
...
-						Please let us know when you have received your laptop / when you
-						have picked up your laptop at{" "}
-						<strong>
+						Please let us know
 							{request.laptopAssignment.deliveryOption === "ship"
-								? request.requestAddress
-								: request.laptopAssignment.address}
-							.
-						</strong>
+								? " when you have received your laptop " + request.requestAddress
```
Comment (shieldo) [inline client/src/pages/ViewRequestStatus/RequestStatusView.js]: I think the wording on this line maybe needs an "at" before the address?

**Review by shieldo — COMMENTED** [review]
Looking generally fine - just a very small comment not related to code logic.

File: server/db_setup.sql [hunk 2]
```diff
@@ -1,11 +1,15 @@
 DROP Table IF EXISTS laptop_request, laptop_donation, laptop_assignment;
-DROP TYPE IF EXISTS assignment_status;
+DROP TYPE IF EXISTS assignment_status, laptop_request_status;
+
+CREATE TYPE laptop_request_status AS ENUM ('ACTIVE', 'CANCELED');
```
Comment (shieldo) [inline server/db_setup.sql]: To be consistent with existing uses the British spelling `CANCELLED` should probably be used here.

Comment (Fareedh-12) [reply server/db_setup.sql]: Done, sorry for not keeping an eye out for this

**Review by gregdyke — APPROVED** [review]

**Pull request merged**